<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TweaksSettingsView</class>
 <widget class="QWidget" name="TweaksSettingsView">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>TWEAKS</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="TouchLabel" name="homeMenuLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24px Georgia;
border-top: 1px solid black;</string>
     </property>
     <property name="text">
      <string>Home Menu</string>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TouchLabel" name="miscLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24px Georgia;
border-top: 1px solid black;</string>
     </property>
     <property name="text">
      <string>Miscellaneous</string>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <widget class="TouchLabel" name="ledLabel">
     <property name="minimumSize">
      <size>
       <width>0</width>
       <height>60</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>60</height>
      </size>
     </property>
     <property name="styleSheet">
      <string notr="true">font: 24px Georgia;
border-top: 1px solid black;</string>
     </property>
     <property name="text">
      <string>LEDs</string>
     </property>
     <property name="indent">
      <number>10</number>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>417</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>TouchLabel</class>
   <extends>QLabel</extends>
   <header>TouchLabel.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
